wrappers/ios/Sources/Wrapper/Writer/ZXIBarcodeWriter.mm

```diff
@@ -18,8 +18,20 @@
                         sizeof(wchar_t));
 }
 
+std::wstring NSDataToStringW(NSData *data) {
+    std::wstring s;
+    const unsigned char *bytes = (const unsigned char *) [data bytes];
+    size_t len = [data length];
+    wchar_t buf[len];
+    for (int i = 0; i < len; ++i) {
+        buf[i] = bytes[i];
+    }
+    s.assign(buf, len);
+    return s;
```

Comment:
Minor cosmetic: a somewhat cleaner version without the `buf` would be along the lines of:
```
			std::wstring s;
			for (int i = 0; i < len; ++i)
				s.push_back(bytes[i]);
```